atomic_reactor/utils/remote_host.py

```diff
@@ -653,9 +653,7 @@ def from_config(cls, config: dict, platform: str):
                 hostname=hostname, username=attr["username"], ssh_keyfile=attr["auth"],
                 slots=attr.get("slots", 1), socket_path=attr["socket_path"], slots_dir=slots_dir
             )
-            # Check whether host is operational before use it
-            if host.is_operational:
-                hosts.append(host)
```

Comment:
We would still need to `mkdir` the slots dir (is_operational does that as a side effect)

Reply:
you can also change is_operational and make there few retries with small delay .....

Reply:
I actually moved `is_operational` to call just before `available_slots` so that way we immediately know if a machine is operational or not right before checking the slots

Reply:
Seems reasonable :+1: